kolibri/plugins/management/assets/src/vue/class-edit-page/index.vue

```diff
@@ -23,18 +23,17 @@
     </div>
 
     <div class="toolbar">
-      <div class="searchbar" role="search">
+      <div class="searchbar" :class="{'searchbar-mobile': elSize.width < 600}" role="search">
```

Comment:
> I'd love to have a relatively small set of constants to use from for element query if we know what these constants are.

That's a good idea. Not sure how we would choose those but it might be convenient.

[Material specifies](https://material.io/guidelines/layout/metrics-keylines.html#) that "All components align to an 8dp square baseline grid for mobile, tablet, and desktop" so maybe theres a way of ensuring that all element query breakpoints are multiples of 8.

> the side-menu causes a situation where the window size not changed but the element size gets changed a lot. That's why breakpoints not suitable here.

It's true that the element size changes when the nav bar opens at larger screen sizes. I'm just following [Material's guidelines here](https://material.io/guidelines/layout/responsive-ui.html#responsive-ui-breakpoints). It shows that the page content gets "squished" when the nav bar opens rather than reflowing.

I should mention that I initially implemented another page exactly like you did for the exact same reason until I read the guidelines in more detail! Maybe that's actually the right way to do it - but if we want to do it that way, we should try to be consistent across all pages.